build.gradle

```diff
@@ -17,7 +17,10 @@ dependencies {
     runtimeOnly 'mysql:mysql-connector-java'
 	runtimeOnly 'com.h2database:h2'
     testImplementation('org.springframework.boot:spring-boot-starter-test')
-
+    compileOnly 'org.projectlombok:lombok:1.18.16'
```

Comment:
Theres been updates to master that added Lombok to the project. You might want to pull in those updates and resolve the conflict.

Reply:
Done!

Reply:
Resolved the conflict

Reply:
Still see the build failing here. It shows that one your tests is failing.

Reply:
resolved and build is successful.